sample-configs/run_config_mediapipe_facemesh.json

```diff
@@ -0,0 +1,36 @@
+{
+    "benchmarkName": "mediapipe facemesh",
+    "frameworksToTest": ["tensorflow.js", "ort-web"],
+    "backendsToTest": ["wasm", "webgl"],
+    "warmupIteration": 5,
+    "runIteration": 25,
+    "logLevel": "error",
+    "debug": false,
+    "ortweb": {
```

Comment:
don't see ort shape info here. will it share the information with tfjs config?

Reply:
This model has a shape with fixed dimensions. In this case, a tool reads input shape information from a onnx model and a tfjs model automatically.

Reply:
tfjs shape is not required, either. But, I intentionally put a shape for an example.

Reply:
thanks for clarification.